components/builder-web/app/actions/projects.ts

```diff
@@ -13,12 +13,28 @@
 // limitations under the License.
 
 import * as fakeApi from "../fakeApi";
-import {Observable} from "rxjs";
-import {BuilderApiClient} from "../BuilderApiClient";
-import {addNotification} from "./notifications";
-import {DANGER, INFO, SUCCESS, WARNING} from "./notifications";
-import {requestRoute, resetRedirectRoute} from "./router";
-import {packageString} from "../util";
+import {
```

Comment:
I think this is the prefered style: https://angular.io/guide/styleguide#import-line-spacing. We should try to get the linter to enforce that later.

Reply:
ah sorry, ran an auto beautifier